README.adoc

```diff
@@ -117,6 +117,27 @@ You should see the following output in your terminal:
 <a href="http://asciidoctor.org"><strong>Asciidoctor</strong></a> running on <a href="http://opalrb.org"><em>Opal</em></a> brings AsciiDoc to Node.js!</p>
 ....
 
+## Node.js examples
```

Comment:
I think the title is misleading because the JavaScript produced is interpreted in a Browser, maybe "Running examples" ? or just "Examples" ?